File: CS/CodeBehind/EFCore/InfiniteAsyncSource/Issues/Issue.cs
using System;

namespace EFCoreIssues.Issues {
    public class Issue {
        public int Id { get; set; }
        public string Subject { get; set; }
        public int UserId { get; set; }
        public virtual User User { get; set; }
        public DateTime Created { get; set; }
        public int Votes { get; set; }
        public Priority Priority { get; set; }
        public Issue() {
            Created = DateTime.Now;
        }
    }
    public enum Priority { Low, BelowNormal, Normal, AboveNormal, High }
}

File: VB/CodeBehind/EntityFramework/PagedAsyncSource/Issues/Issue.vb
Namespace Issues
    Public Class Issue
        Public Property Id As Integer
        Public Property Subject As String
        Public Property UserId As Integer
        Public Overridable Property User As User
        Public Property Created As Date
        Public Property Votes As Integer
        Public Property Priority As Priority

        Public Sub New()
            Created = Date.Now
        End Sub
    End Class

    Public Enum Priority
        Low
        BelowNormal
        Normal
        AboveNormal
        High
    End Enum
End Namespace

